### PR TITLE
chore(ui): add what's new content for 1.2.0

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
### Describe your changes:

I worked on the what's new content for 1.2

https://github.com/open-metadata/OpenMetadata/assets/59080942/f8009e3e-506d-4eea-8d11-92f77c5657c4

#
### Type of change:

- [x] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/ChangeLogs.tsx [hunk 1]
```diff
@@ -25,10 +25,15 @@ const ChangeLogs = ({ data }: Props) => {
 
   return (
     <div>
-      {logKeys.map((log, index) => (
-        <div className="mb-4" key={index}>
+      {logKeys.map((log) => (
+        <div className="mb-4" key={log}>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/ChangeLogs.tsx:29]: Don't we have any unique identifier here?

**Review by chirag-madlani — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/ChangeLogs.tsx (hunk 1 above)
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/ChangeLogs.tsx:29]: That log is a unique key, so that will work.

Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/ChangeLogs.tsx:29]: Hy we can't have longer text as the key
Will log always be of few words?

Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/ChangeLogs.tsx:29]: Yes, the log will be of a few words only.

**Pull request merged**